### ISSUE TITLE
Feature request - possibility to specify protocol version

### ISSUE THREAD

**Issue by CyberGuerro** [aliasadidev/vscode-npm-gui#49]
Is it possible to specify the protocol version es. v2 instead of v3 (default).
My request is based of needs to use DevExtreme repository which use v2 protocol (output returned is in xml format instead of JSON) and when I send request it returns me an xml file that isn't parsable returning an error like "invalid JSON format".

**Comment by aliasadidev** [comment]
you're right, this extension doesn't support version v2 (it's a little old honestly), 
I will check that

**Comment by CyberGuerro** [comment]
Any news? It would be very useful have it

**Comment by CyberGuerro** [comment]
Question:     Development is dead??
It's long long time that no updates are released.
